build.gradle

```diff
@@ -26,13 +26,15 @@ dependencies {
     compileOnly 'net.luckperms:api:5.4'
     compileOnly 'dev.simplix:protocolize-api:2.2.5'
     compileOnly 'io.netty:netty-codec-http:4.1.89.Final'
+    compileOnly 'io.github.miniplaceholders:miniplaceholders-api:2.0.0'
     compileOnly 'net.william278:PAPIProxyBridge:1.0'
     compileOnly 'org.projectlombok:lombok:1.18.26'
 
     implementation 'org.apache.commons:commons-text:1.10.0'
     implementation 'net.william278:Annotaml:2.0.1'
     implementation 'dev.dejvokep:boosted-yaml:1.3.1'
     implementation 'de.themoep:minedown-adventure:1.7.2-SNAPSHOT'
+    implementation 'net.kyori:adventure-text-minimessage:4.12.0'
```

Comment:
MiniMessage is already included in Velocity 3.1.2 and higher, which more than 95% of proxies use (according to bstats)

Reply:
Oh, right. The latest stable API is 3.1.1, which doesn't bundle MiniMessage, so I thought it needed shading. I'll just move it to compileOnly.